### DISCUSSION TITLE
Creating a founding text for the group

### DISCUSSION THREAD

**Discussion by mrchristian** [singlesourcepub/community#6] [General]
Hi,

Is this the document we should use as the source for creating our group launch text? https://writemd.rz.tuhh.de/oG0hzswrT7iuHGatwnH_LA

If so what do we think is the best workflow to develop it into the 'launch text'?

And I would suppose linked to this question would be what GitHub Pages website workflow do we want to follow, Hugo based I assume?

Simon

**Comment by mrchristian** [comment]
I'll have some time over Thursday, Friday to work on the text: I figure we need:

- Mission
- Announce
- Website basics - UX Personas and branding: About; CoC or some sorts; Imprint, ?
- Repo basics: licence; contribute;

**Reply by xldrkp** [reply]
Thanks for your thoughts. I will try to join you Thursday and Friday on the texts. Let's use the pad mentioned as a starting point.

**Reply by mrchristian** [reply]
-good one - What do you prefer for writing´, shall we stick with pads and then port to GitHub, there is a Gist workflow - not always 100% on top of that, but like idea.

Can we fork that pad in Hedge Doc and make a collection of docs?

Well for now and long as we have an address for the doc, where others can see whats happening and take part, that will do

**Comment by mrchristian** [comment]
It will now be Saturday as a slot for me to work on this. A book needs a proofing overhaul and another one needs cover artwork and booksheet text - as is the publishing life :-)

**Comment by mrchristian** [comment]
Finally I ahve a bit of time . I'll look through our existing texts today and then get started

**Reply by xldrkp** [reply]
As the free pads would be free forever and editable for the world I decided to go with the wiki here. I think this part of the the work can be publicly readable so that everybody can join the team and help writing. And I would not go with pull requests and the like right now, writing in the wiki is easy.

**Reply by xldrkp** [reply]
I've cleaned up everything here a little, reduced the project functionaliy and set up the wiki for writing the founding texts for the group. Finally, I renamed the project to community.

**Reply by mrchristian** [reply]
This a great approach. I've booked myself in 13-16 today to do some writing :-)

**Comment by mrchristian** [comment]
Name? Is it:

Single Source Publishing Community

SSPC

or

Scholarly Single Source Publishing Community 

SSSPC

Would we shorten down either and refer to ourselves as the Single Source Community.

Searching sspc and ssspc I dont see any obvious ploblems

**Comment by mrchristian** [comment]
I'm looking for frameworks for how we prioritise ideas and challenges we see in our sector of SSP for the process of Mission building. I have brought together some frameworks or methods for other areas - but not this one - any suggestions? See https://github.com/singlesourcepub/community/wiki/Organisation

**Comment by mrchristian** [comment]
@xldrkp I have a bullet point outline here for the announce https://github.com/singlesourcepub/community/wiki/Announcement - easily fill 800 words :-) @xldrkp do you have a SSP illustration and maybe a second one on Open Science challenges and SSP? Just a thought.

**Comment by mrchristian** [comment]
I've now got the main body of an annoucne and blog post ready for SSPC @xldrkp https://github.com/singlesourcepub/community/wiki/Announcement-Blog#text its at 525 words and we should keep it under 800 words.

I thought I'd leave it here and hand over to you all as I think I've communicated the launch of the community, but also I'm assuming a lot, which needs some feedback, and to leave space for others.

I'll now look at what else we need: basic website, other bits and bobs :-) Lets get the show on the road.

**Comment by mrchristian** [comment]
I got quite far today, and with some time on things tomorrow morning I can get things into a state where they can be reviewed in consideration for launching.

I've tried to keep it simple. Tried, honest :-) But we needed some basics.

1. One pager website with translation in Hugo, Imprint, data protection - To Do
2. Code of Conduct - I've used a simple version from The Turing Way as guidance with no enforcement - besides telling someone to stop.
3. Membership - What is it, how do you become a member. Happy to reconsider in any way, but I made it two conditions, you agree to CoC and offer some help. Then add your name to list?
4. Mission - done without any goals yet. But think not needed at moment.
5. Announce / Blog - Draft ready, can have another 200 words added and needs graphics.
6. Imprint, data protection pages

**Comment by mrchristian** [comment]
@xldrkp you had a chance to look at blog and other texts, I can review and proof again first thing tomorrow. Would be good to get out Friday :-)
